content/zh/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -0,0 +1,222 @@
+---
+title: 查看 pod 和节点
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+
+<div class="layout" id="top">
+
+  <main class="content">
+
+    <div class="row">
+
+      <div class="col-md-8">
+<!--
+        <h3>Objectives</h3>
+-->
+        <h3>目标</h3>
+        <ul>
+<!--
+          <li>Learn about Kubernetes Pods.</li>
+          <li>Learn about Kubernetes Nodes.</li>
+          <li>Troubleshoot deployed applications.</li>
+-->
+          <li>了解 Kubernetes Pod.</li>
+          <li>了解 Kubernetes Node.</li>
+          <li>对已部署的应用故障排除</li>
+        </ul>
+      </div>
+
+      <div class="col-md-8">
+<!--
+        <h2>Kubernetes Pods</h2>
+-->
+        <h2> Kubernetes Pod </h2>
```

Comment:
this is in HTLM code :) no need to space between the tag and words
```suggestion
        <h2>Kubernetes Pods</h2>
```